lemmy/.env-dist

```diff
@@ -0,0 +1,32 @@
+# The domain name for the whoami service:
+LEMMY_TRAEFIK_HOST=lemmy.example.com
+LEMMY_VERSION=0.18.3
+LEMMY_UI_VERSION=0.18.3
+ASONIX_PICTRS_VERSION=0.4.1
+
+# The name of this instance. If there is only one instance, use 'default'.
+LEMMY_INSTANCE=
+
+# You can degine the port that Traefik uses for the lemmy service.
+LEMMY_NIGNX_PORT=8536
```

Comment:
misspelled. Also, I don't think this needs to be exposed.

Reply:
It's static in the nginx.conf so it cant be configured otherwise.

Reply:
fixed in [494d31e](https://github.com/EnigmaCurry/d.rymcg.tech/pull/92/commits/494d31e20d60eddb535e1834d1b8b4e8a65d0d3e)